src/components/themed/TransactionListTop.tsx

```diff
@@ -794,7 +804,8 @@ export function TransactionListTop(props: OwnProps) {
       theme={theme}
       walletName={walletName}
       walletStakingState={walletStakingState}
```

Comment:
The only remaining things we read from `walletStakingState` are `stakePositionMap` and `lockedBalance`. So please just pass these in directly, and get rid of the big state object.

If you do this, you will find that `stakePolicies` are still being read from the `walletStakingState` instead of your new prop - so potentially more crash locations.